ui/pages/notifications/notification-components/types/notifications/notifications.ts

```diff
@@ -52,7 +71,13 @@ export type NotificationComponent<N extends Notification = Notification> = {
   item: NotificationFC<N>;
   details: {
     title: NotificationFC<N>;
-    body: BodyFeatureAnnouncement<N> | BodyOnChainNotification<N>;
+    body:
+      | BodyFeatureAnnouncement<N>
+      | BodyOnChainNotification<N>
+      | BodySnapNotification<N>;
   };
-  footer: FooterFeatureAnnouncement<N> | FooterOnChainNotification<N>;
+  footer:
+    | FooterFeatureAnnouncement<N>
+    | FooterOnChainNotification<N>
+    | FooterSnapNotification<N>;
```

Comment:
Nice. I think our team might do another refactor in the future to better group these unions.
E.g. it is unlikely that you would have a FeatureAnnonucement Body, but a Snap Footer.

IDK why we didn't group the footer in the `details` section 😛 

Noted and will work on this later.